spec/requests/api/v1/api_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe RedeemifyCodesController, type: :request do
+  describe "GET /users/:user_id/redeemify_codes/:code" do
```

Comment:
great!  We could also use named subject, e.g. 

```rb
subject(:make_get_request) { get '/redeemify_codes/:id', format: :json, params: { id: @rcode.code } }
```

see https://www.relishapp.com/rspec/rspec-core/docs/subject/explicit-subject